interface/main/tabs/main.php

```diff
@@ -1,5 +1,6 @@
 <?php
-Use Esign\Api;
+use Esign\Api;
+use OpenEMR\Core\Header;
```

Comment:
And if we don't do this, we need to keep it standard (ie. do one way or the other).